FormalConjectures/ForMathlib/Order/Interval/IsInterval.lean

```diff
@@ -0,0 +1,24 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import Batteries
+
+universe u v
+
+/-- The predicate that a set `A` is an interval, i.e. for all `a, b` in `A`, if `a ≤ c ≤ b` then
+`c` also lies in `A`. -/
+def IsInterval {α : Type u} {σ : Type v} [LE α] [Membership α σ] (A : σ) : Prop :=
+  ∀ᵉ (a ∈ A) (b ∈ A) (c), a ≤ c → c ≤ b → c ∈ A
```

Comment:
This seems quite similar to `Set.OrdConnected` (which has some amount of API already). Is it possible to use `OrdConnected` instead here?